splice_machine/src/main/java/com/splicemachine/derby/impl/sql/execute/actions/TruncateTableConstantOperation.java

```diff
@@ -225,9 +226,12 @@ private void truncateTable(TableDescriptor td, Activation activation) throws Sta
             }
         }
 
+        // Remove statistics
+        dd.deleteColumnStatistics(oldHeapConglom, tc);
+        dd.deletePartitionStatistics(oldHeapConglom, tc);
```

Comment:
DataDictionaryImpl.deletePartitionStatistics() will delete both table and column stats, so `dd.deleteColumnStatistics(oldHeapConglom, tc);` is redundant.

Reply:
Done.